src/screens/admin/FieldMappingConfig.tsx

```
@@ -0,0 +1,1052 @@
+	getMapping,
+	updateMapping,
+	fetchFields as fetchFieldsAPI,
+} from '../../services/admin/admin';
+import Layout from '../../components/common/admin/Layout';
+
+// Props interface for FieldMappingConfig
+interface FieldMappingConfigProps {
+	refreshTrigger?: number;
+}
+interface VcFieldValue {
+						type: string;
```

ESLint

[error] Delete `↹↹↹↹↹`

(prettier/prettier)